tests/unit/defrag.cpp

```diff
@@ -566,3 +566,53 @@ BOOST_AUTO_TEST_CASE(test_defrag_in_dir) {
 	// check that defrag succeeded
 	BOOST_REQUIRE_EQUAL(eblob_stat_get(backend->stat, EBLOB_GST_DATASORT_COMPLETION_STATUS), 0);
 }
+
+
+/**
+ * Fill two blobs and insert one item.
+ * Delete a half from each blob and defrag them.
+ * Finally, check number of touched bases in blob, sizes of live/remove data,
+ * number of live/removed records.
+ */
+BOOST_AUTO_TEST_CASE(defrag_stats) {
```

Comment:
couldn't we check stat correctness on each test in `datasort` as check_view_usage is doing?

Reply:
It's not so easy because we invoke directly generate_sorted_data there but defrag stats initialized at defrag_in_dir which supposed to to be invoked in a separate thread but we disable it in tests. Am I explaining too difficult?)

Reply:
nobody can prevent you from zeroing those stats explicitly as it is done in check_view_usage )

Reply:
done